### PR TITLE
Typed version of the ForkJoinActorBenchmark

### PR DESCRIPTION
**Pull request by johanandren**
Refs #25986

### CODE REVIEW

File: akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedForkJoinActorBenchmark.scala [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  final val messages = 2000000 // messages per actor pair
+
+  val timeout = 30.seconds
+
+  // Constants because they are used in annotations
+  // update according to cpu
+  final val cores = 8
+  final val coresStr = "8"
+  final val cores2xStr = "16"
+  final val cores4xStr = "24"
```
Comment (patriknw) [inline akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedForkJoinActorBenchmark.scala]: 32?

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedForkJoinActorBenchmark.scala (hunk 1 above)
Comment (johanandren) [reply akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedForkJoinActorBenchmark.scala]: That would make sense but just mindless copy pasta of the numbers in the original one here.

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**